packages/client/src/pages/LeaderBoard/components/HighscoreList/highscoreList.tsx

```diff
@@ -1,24 +1,73 @@
-import React from 'react';
+import React, { useCallback, useEffect, useRef } from 'react';
+import { LeaderboardRequest } from '@models/api/leaders';
+import { ButtonVariant } from '@components/Button';
 
-import s from './highscoreList.module.scss';
+import { useAppDispatch, useAppSelector } from '@/store/hooks';
+import { getLeaders } from '@/store/reducers/leaders/leadersActionCreator';
+import { Button, Spacer, Text } from '@/components';
 
-const mockHighscores = [
-	{ username: '#playerName1', score: 19000 },
-	{ username: '#playerName2', score: 1920 },
-	{ username: '#playerName3', score: 1900 },
-	{ username: '#playerName4', score: 800 },
-	{ username: '#playerName5', score: 500 }
-	// todo:заменить моковые данные на данные из бэка
-];
+import s from './highscoreList.module.scss';
 
+// todo: добавить нормальную развернутую пагинацию/фильтрацию
```

Comment:
_Не критично к исправлению_

В реальном проекте такой `todo` навсегда затеряется и, скорее всего, никогда так и не будет реализован. Более предпочтителен вариант сразу завести задачу на доделку/фикс и указать тут уже ссылку на саму задачу или же номер задачи

```suggestion
// todo: добавление нормальной развернутую пагинации/фильтрации в `YA-1234`
```